spec/message.abnf

```diff
@@ -32,11 +32,9 @@ annotation            = function
                       / private-use-annotation
                       / reserved-annotation
 
-; Markup; Note that standalone markup is part of the first line
-markup       = "{" [s] markup-open *(s attribute) [s] ["/"] "}"
-             / "{" [s] markup-close *(s attribute) [s] "}"
-markup-open  = "#" identifier *(s option)
-markup-close = "/" identifier
+markup = "{" [s] "#" identifier *(s option) *(s attribute) [s] "}"  ; open
+       / "{" [s] "#" identifier *(s option) *(s attribute) [s] "/}" ; standalone
```

Comment:
```suggestion
markup = "{" [s] "#" identifier *(s option) *(s attribute) [s] ["/"] "}"  ; open and standalone
```

Reply:
There's precedent in our ABNF of refactoring productions to make them more readable for humans at the cost of increasing the lookahead. However here keeping `open` and `standalone` as separate alternatives results in `LL(infinity)` due to `*` on options and attributes. I'd prefer to not go that far, so I agree with @eemeli's suggestion.